README.md

```diff
@@ -1,28 +1,87 @@
 # Modelz Python SDK
 
+[Docs](https://tensorchord.github.io/modelz-py/) | [Templates](https://cloud.modelz.ai/templates) | [ModelZ](https://modelz.ai) | [ModelZ Docs](https://docs.modelz.ai/)
+
+[ModelZ](https://modelz.ai) is an MLOps platform, you can deploy serverless instance for machine learning models by packed Docker image, such as Stable Diffusion, Dolly, ImageBind, and so on...
+
+`Deployment` is an instance of any ML service deployed at `ModelZ`, you could create one and then make `inference` to it. 
+
+`Templates` are preset Docker images for `deployment`, which is widely acknowledged used ML models, official templates are built and maintained by `ModelZ` developers. While it's available for you to define your own `template` and `deployment`.
+
+The python SDK is designed for CURD to your `deployments`, and sent request to them to make `inference`. It 's an alternative of ModelZ WebUI operation, which could be more friendly with CI/CD pipelines or at model development.
+
+## Install
+
 ```shell
 pip install modelz-py
 ```
 
-## CLI
+## CLI usage
+
+We support these functions now:
+
+- create/update/list/delete deployment
+- make inference to deployments
+- get metric information of any deployment
+
+This functions will be supported in the future:
+
+- build image and push to registry
+
+**See [CLI Docs](https://tensorchord.github.io/modelz-py/cli.html) for all usages.**
+
+## Example
+
+### Create and infer to ModelZ deployment at terminal
+
+#### Step 1: Create deployment
+First, you need to create a deployment at ModelZ platform. We pick `Stable Diffusion` image for this example.
+To get more predefined images, see our [templates](https://cloud.modelz.ai/templates).
+
+```shell
+export MODELZ_API_KEY=mzi-1234567890987654321
+export MODELZ_USER=00000000-1111-1111-1111-000000000000
+modelz deployment create \
+--image us-central1-docker.pkg.dev/nth-guide-378813/modelzai/mosec-stable-diffusion:23.04.1 \
+--server-resource nvidia-tesla-t4-4c-16g \
```

Comment:
Where user can get the image name and server resource name?

Reply:
Add help message for image name.
server resource are generated fixed value, and could be get from command help.